php/admin-functions-term.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
A lot of the functions in this file do not have a `$term_id` parameter.  I just wanted to make sure that this is intentional.